cypress/integration/ResourcesForResearchers.feature

```diff
@@ -0,0 +1,229 @@
+        And the search result description is displayed
+        And both pagers are displayed
+
+    Scenario: User is able to see the No results found page
+        Given screen breakpoint is set to "desktop"
+        Given user is navigating to "/research/resources"
+        When user types "fre" in the search bar
+        When user hits "enter" key
+        Then user is redirected to "/research/resources/search" with "?from=0&q=fre"
+        And page title is "Resources for Researchers Search Results"
+        And search box displays the term "fre"
+        And the message is displayed as "No results were found for your search. Please try another search or view all resources."
+        And "view all resources." is a link with href "#viewAll"
+        And the following "<pageOption>" are displayed
+            | pageOption |
+            | Twitter    |
+            | Email      |
+
+    Scenario Outline: User is able to see the Resource Detail page
+        Given screen breakpoint is set to "desktop"
+        Given user is navigating to "<url>"
+        Then the resource title is displayed as "<title>"
+        And "Resources for Researchers Home" is displayed with href "/research/resources/"
+        And "Resource description" field is displayed
+        And Visit Resource link is displayed
+        And search box is displayed
+        And "Find Related Resources" section is displayed
+        And "Resource Access Information" field is displayed
+        And "NCI Affiliation Information" field is displayed
+        And Contact Information field is displayed
+
+        Examples:
+            | url                              | title                                                 |
+            | /research/resources/resource/82  | Health Disparities Calculator (HD*Calc)               |
+            | /research/resources/resource/102 | Allele-Specific Alternative mRNA processing (ASARP)   |
+            | /research/resources/resource/32  | The International Registry of Werner Syndrome         |
+            | /research/resources/resource/54  | Cancer Intervention and Surveillance Modeling Network |
+
+
+    Scenario: User is able to go to Resource Detail page from Search Results page
+        Given screen breakpoint is set to "desktop"
+        Given user is navigating to "/research/resources"
+        When user clicks on "Cancer Biology"
+        Then user is redirected to "/research/resources/search" with "?from=0&researchAreas=cancer_biology"
+        And the filter displays the "Cancer Biology"
+        And search box is empty
+        And results info text is "1 - 20 of \d+ results"
+        And the "Research Area" box displays the filter "Cancer Biology" as checked
+        When user clicks on 1 search result item
+        # Then the page title is the same as the clicked result's item title
+        And the "Back to Search Results" is displayed
```

Comment:
this should be 
 And the "< Back to Results" is displayed

Reply:
done